app/src/CategoryRow.jsx

```diff
@@ -85,15 +85,33 @@ export default function CategoryRow(props) {
   }
 
 
-  function overpassLink(k, v, n) {
-    const q = encodeURIComponent(`[out:json][timeout:60];
-(nwr["${k}"="${v}"]["name"="${n}"];);
+  function overpassLink(k, v, n, w) {
+    // Build Overpass Turbo link:
+    const q = encodeURIComponent(`[out:json][timeout:100];
+(nwr["name"="${n}"];);
 out body;
 >;
-out skel qt;`);
+out skel qt;
+
+{{style:
+node[name=${n}],
+way[name=${n}],
+relation[name=${n}]
+{ color:red; fill-color:red; }
+node[${k}=${v}][name=${n}],
+way[${k}=${v}][name=${n}],
+relation[${k}=${v}][name=${n}]
+{ color:yellow; fill-color:yellow; }
+node[${k}=${v}][name=${n}][brand=${n}][brand:wikidata=${w}],
+way[${k}=${v}][name=${n}][brand=${n}][brand:wikidata=${w}],
+relation[${k}=${v}][name=${n}][brand=${n}][brand:wikidata=${w}]
+{ color:green; fill-color:green; }
+}}`);
+    
+    // Create Overpass Turbo link:
     const href = `https://overpass-turbo.eu/?Q=${q}&R`;
     return (
-      <a target="_blank" href={href}>View on Overpass Turbo</a>
+      <a target="_blank" href={href} title="View ${n} via Overpass Turbo">View ${n} via Overpass Turbo</a>
```

Comment:
Template string syntax doesn’t work here; this line is JSX rather than JavaScript. 5455bc9be136b55f39cb408fd94969e667f7391b should fix the stray dollar signs.